code/game/objects/buckling.dm

```diff
@@ -20,6 +20,17 @@
 			if(user_unbuckle_mob(buckled_mobs[1],user))
 				return 1
 
+/atom/movable/attackby(obj/item/W, mob/user, params)
+	if(!can_buckle || !istype(W, /obj/item/riding_offhand))
+		return ..()
```

Comment:
Looking at this condition again now--what happens if you attack a chair? Will check to make sure it's normal.

Reply:
It works fine, but I don't know if that's just a lucky coincidence based on the ordering.